fastapi/routing.py

```diff
@@ -41,6 +42,7 @@
 from pydantic.error_wrappers import ErrorWrapper, ValidationError
 from pydantic.fields import ModelField, Undefined
 from starlette import routing
+from starlette.applications import Starlette
```

Comment:
You only use this import for Typing purposes, wouldn't it make more sense to make use of the more general `ASGIApp` type in `starlette.types`?

Reply:
In `starlette.routing` the type used is Any.